package.json

```diff
@@ -187,13 +158,22 @@
     "@types/node": "^12.12.0",
     "@typescript-eslint/eslint-plugin": "^4.23.0",
     "@typescript-eslint/parser": "^4.23.0",
+    "chai": "^4.3.6",
+    "chokidar": "^3.5.3",
```

Comment:
I would think it's not really a "devDependencies". Though we don't publish npm module and the production code is bundled so it's not really a problem.
I would suggest move it to the "dependencies" part in the "client/package.json" file. So it's clear that it's used in the production code in client. We keep client and server clear so they can be separated when needed.

Reply:
Good suggestion. Updated.